src/Core/src/Platform/iOS/LayoutView.cs

```diff
@@ -37,11 +41,10 @@ public override void WillRemoveSubview(UIView uiview)
 				return null;
 			}
 
-			if (!result.UserInteractionEnabled)
+			if (Subviews.Contains(result) && !result.UserInteractionEnabled)
```

Comment:
Would it be better to rather do `result.Superview == this` instead of getting the subviews array and looping through them all?

Reply:
Also, I am thinking we can't just check the result, we need to find the parent that _is_ the child of the layout. So I am thinking:

```cs
while (result.Superview is not null && result.Superview != this)
    result = result.Superview;
if (result.Superview == this && !result.UserInteractionEnabled)
```

If we have a secret scroll view in our webview, we still need to find that webview and read the property the user set in XAML.